packs/spell-effects/aura-bane.json

```diff
@@ -0,0 +1,58 @@
+{
+    "_id": "F5NnpKkn4pKwEeFB",
+    "img": "systems/pf2e/icons/spells/bane.webp",
+    "name": "Aura: Bane",
```

Comment:
We generally don't have reminder-only spell effects. See:
https://github.com/foundryvtt/pf2e/pull/11374
https://github.com/foundryvtt/pf2e/pull/15529

Reply:
I see, but we're nearly a year on with no current save support for auras yet. This is more useful having than not having , https://github.com/foundryvtt/pf2e/pull/11374#issuecomment-1813126768 while I get this its just as true the opposite. I included both effects in banes description.

The reason for this PR was mainly because both Bless and Bane are in a weird state. I wanted them to match up. 

People complain Bane does not have an aura, you explain why then they have to create custom aura and edit items etc.
People complain the Bane aura does nothing, you explain why then all they have to do is apply the effects.

Reply:
People complain about a lot of things, but that doesn't change our stance that we don't want tracking effects.

Reply:
Okay do you want to delete the Aura effect then?

I don't see as this as simple reminder effect though, it shows players the size of the aura and how many times its been sustained making it easier in game for knowing when to roll saves. Having Bane in its current state when Bless is essessentially complete is jarring, Bane looks incomplete/broken.

Their are also plenty of Aura effects in the system that are their for tracking the Aura area for an effect I don't see how this is any different.

Reply:
Yeah, just remove it for now.